bindata/cloud-network-config-controller/controller.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cloud-network-config-controller
+  namespace: openshift-cloud-network-config-controller
+  annotations:
+    kubernetes.io/description: |
+      This deployment launches the cloud network config controller which manages cloud-level network configurations
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+    networkoperator.openshift.io/non-critical: ""
+spec:
+  selector:
+    matchLabels:
+      app: cloud-network-config-controller
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      name: cloud-network-config-controller
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: cloud-network-config-controller
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      serviceAccountName: cloud-network-config-controller
+      priorityClassName: system-cluster-critical
+      containers:
+      - name: controller
+        image: {{.CloudNetworkConfigControllerImage}}
+        imagePullPolicy: IfNotPresent
+        command: ["/usr/bin/cloud-network-config-controller"]
+        args: ["-cloud-provider", "{{.CloudProvider}}", "-cloud-region", "{{.CloudRegion}}", "-secret-name", "cloud-credentials"]
```

Comment:
No, on Azure it will be an empty string, but that's not a problem: the cloud region is only required on AWS for initiating a client when communicating with the cloud API. It can be left empty on Azure.